plugins/arclite/plugin.cpp

```diff
@@ -702,8 +702,9 @@ class Plugin
       });
       const auto error_log = std::make_shared<ErrorLog>();
       size_t im = 0, nm = matched_indices.size();
+	  std::vector<UInt32> indices;
```

Comment:
This changes the semantics: previously on each iteration the vector was recreated, now it's outside the loop and will accumulate all the values.
If you want to optimize here, add `indices.clear()` to the loop.

Reply:
u ar absolutely right, I have changed original behavior function.
fix it here https://github.com/FarGroup/FarManager/commit/d832463aa807c429ca15719bcc7cb5939f306fbd